arches/app/media/js/viewmodels/workflow-step.js

```diff
@@ -227,6 +236,12 @@ define([
             }
         }
 
+        this.setSourceStepLock = function(locked) {
+            self.lockableExternalSteps().forEach(function(step){
```

Comment:
I see `self.lockableExternalSteps` being invoked here, but I'm not seeing where it's declared on this level? Can we do something similar to `externalStepData`, where we set the value here based on the config?